### ISSUE TITLE
SUMA 4.1 docs on doc.suse.com have lost their Canonical URL <link> tag

### ISSUE THREAD

**Issue by sknorr** [uyuni-project/uyuni-docs#1031]
## Documentation Update Required

Since the update to the SUMA 4.1 docs that @keichwa prepared today, the SUMA 4.1 files lack this kind of tag:

```
    <link rel="canonical" href="https://documentation.suse.com/suma/4.2/suse-manager/upgrade/proxy-uyuni.html">
```

(As you can see, SUMA 4.2 still has such tags--although the path is wrong, #1032.)

## Subject Matter Expert (SME)

@jcayouette / @keichwa 

## Version

SUMA 4.1

**Comment by keichwa** [comment]
I patched it temporarily with a sed hack.

**Comment by keichwa** [comment]
I think we still must backport the Makefile hack to `manager-4.1`.

**Comment by sknorr** [comment]
I dunno ... but since 4.1 was not translated, I think you need something else there. In fact, 4.1 was fine until the most recent doc update.

**Comment by jcayouette** [comment]
Fixed in 4.1.11. This should no longer be relevent.
